### PR TITLE
[SB] Delete batch changelog

### PR DESCRIPTION
**Pull request by l0lawrence**

### CODE REVIEW

**Comment by l0lawrence** [comment]
need to change version .py

File: sdk/servicebus/azure-servicebus/CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,13 @@
 # Release History
 
-## 7.12.3 (Unreleased)
+## 7.13.0b0 (2024-05-16)
```
Comment (l0lawrence) [inline sdk/servicebus/azure-servicebus/CHANGELOG.md:3]:
```suggestion
## 7.13.0b1 (2024-05-16)
```

File: sdk/servicebus/azure-servicebus/azure/servicebus/_version.py [hunk 2]
```diff
@@ -3,4 +3,4 @@
 # Licensed under the MIT License.
 # ------------------------------------
 
-VERSION = "7.12.3"
+VERSION = "7.13.0b0"
```
Comment (l0lawrence) [inline sdk/servicebus/azure-servicebus/azure/servicebus/_version.py:6]:
```suggestion
VERSION = "7.13.0b1"
```

Comment (swathipil) [inline sdk/servicebus/azure-servicebus/azure/servicebus/_version.py]: b1 here too :)

**Review by swathipil — APPROVED** [review]
lgtm!

**Pull request merged**